src/models/network.js

```diff
@@ -1,5 +1,6 @@
 var _ = require("lodash");
 var Chan = require("./chan");
+var NetworkInfo = require("irc-framework/src/networkinfo");
```

Comment:
Not a fan of requering files like that. I think you can just access the network object in the connect event.